monai/transforms/croppad/dictionary.py

```diff
@@ -232,11 +257,30 @@ def __init__(
     def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
         d = dict(data)
         for key, m in self.key_iterator(d, self.mode):
+            self.push_transform(d, key)
             d[key] = self.padder(d[key], mode=m)
         return d
 
+    def inverse(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        d = deepcopy(dict(data))
+
+        for key in self.key_iterator(d):
+            transform = self.get_most_recent_transform(d, key)
+            # Create inverse transform
+            orig_size = np.array(transform[InverseKeys.ORIG_SIZE.value])
+            current_size = np.array(d[key].shape[1:])
+            roi_start = np.floor((current_size - orig_size) / 2)
```

Comment:
Could you please help add more tests to cover the corner cases that `np.floor()` may shift 1 pixel for the ROI position?

Reply:
I tried to cover all cases by (for each spatial dimension) having odd and even sized input, and cropping/padding by even amounts. On top of that, I have added 1D test cases for all odd/even combinations of input image and amount of cropping/padding.

Reply:
https://github.com/Project-MONAI/MONAI/pull/1737/files#diff-2f0622a3053b0bd6e89b040d75c4abd375d54611a311604b0cbaaa87f9b1d51eR56-R71